mempool/src/shared_mempool/tasks.rs

```diff
@@ -82,7 +82,11 @@ pub(crate) async fn process_client_transaction_submission<V>(
     timer: HistogramTimer,
 ) where
     V: TransactionValidation,
-{
+{ 
+    dbg!("new transaction", &transaction);
+    dbg!("new transaction", &transaction.sender());
```

Comment:
This one could be turned into a prometheus log. Would be interesting to see how many unique users are sending txs per period.

Reply:
I think we already have that metric, or am I mistaken?

```
static SHARED_MEMPOOL_TRANSACTIONS_PROCESSED: Lazy<IntCounterVec> = Lazy::new(|| {
    register_int_counter_vec!(
        "diem_shared_mempool_transactions_processed",
        "Number of transactions received and handled by shared mempool",
        &[
            "status", // state of transaction processing: "received", "success", status code from failed txn processing
            "network", // state of transaction processing: "received", "success", status code from failed txn processing
            "sender"   // sender of the txns
        ]
    )
    .unwrap()
});
```

Reply:
Ok yes thank you.